ibm/service/project/resource_ibm_project.go

```diff
@@ -63,6 +63,11 @@ func ResourceIbmProject() *schema.Resource {
 							Required:    true,
 							Description: "A brief explanation of the project's use in the configuration of a deployable architecture. It is possible to create a project without providing a description.",
 						},
+						"monitoring_enabled": &schema.Schema{
+							Type:        schema.TypeBool,
+							Required:    true,
```

Comment:
Introducing Required field will be a breaking change when user migrate from one version 
Can we mark it as optional and have some default value true/false since its bool

Reply:
yes, the backend code sets it with a default false, I've changed the terraform code as well